scripts/fetch_smugmug_activity.py

```diff
@@ -0,0 +1,262 @@
+from datetime import datetime
+import mysql.connector
+import pytz
+import re
+import requests
+from xml.etree import ElementTree
+
+from scripts.dbconfig import DB_HOST, DB_USER, DB_PASSWORD
+
+
+XML_NAMESPACES = {'atom': 'http://www.w3.org/2005/Atom'}
+
+COMMENT_FEED_URL = 'https://smugmug.com/hack/feed.mg?Type=usercomments&Data=cmubuggy&format=atom10'
+PHOTO_FEED_URL = 'https://smugmug.com/hack/feed.mg?Type=nicknameRecent&Data=cmubuggy&format=atom10'
+
+# pip install mysql-connector-python
+# pip3 install mysql-connector-python-rf
+# Run with 'python3 -m scripts.fetch_smugmug_activity'
+def main():
+    config = {
+        'user': DB_USER,
+        'password': DB_PASSWORD,
+        'host': DB_HOST,
+        'database': 'cmubuggy',
```

Comment:
Database name probably should be also in dbconfig.py like it is in dbconfig.inc.